components/org.wso2.carbon.business.rules.core/src/main/java/org/wso2/carbon/business/rules/core/api/impl/BusinessRulesApiServiceImpl.java

```diff
@@ -166,23 +166,64 @@ public Response getTemplateGroups() throws NotFoundException {
 
     @Override
     public Response loadBusinessRule(String businessRuleInstanceID
- ) throws NotFoundException {
+    ) throws NotFoundException {
         TemplateManagerService templateManagerService = TemplateManagerInstance.getInstance();
         BusinessRule businessRule = templateManagerService.loadBusinessRule(businessRuleInstanceID);
         Gson gson = new GsonBuilder().disableHtmlEscaping().setPrettyPrinting().create();
         return Response.ok().entity(gson.toJson(businessRule)).build();
     }
+
     @Override
     public Response redeployBusinessRule(String businessRuleInstanceID
- ) throws NotFoundException {
+    ) throws NotFoundException {
+        TemplateManagerService templateManagerService = new TemplateManagerService();
+        int status;
+        status = templateManagerService.redeployBusinessRule(businessRuleInstanceID);
```

Comment:
This can be done in a single line

Reply:
Fixed by 2ae9eda4d158d9870c02aa3c46ef96e0cb835e50